### DISCUSSION TITLE
Select elements on the current floor by floor.

### DISCUSSION THREAD

**Discussion by Jamesbdsas** [IfcOpenShell/IfcOpenShell#2832] [Q&A]
How to proceed, select the elements of the current floor by floor.
Could you please advise me how I can find these functions?
Thanks.

**Comment by Moult** [comment]
If you're using Blender, you can right click the collection in the outlinet of the storey and select all elements.

Alternatively you can click on any element on the storey, and in the object properties panel under IFC spatial element, click the icon to select all sibling elements in the same storey.

If you want to do it using python, you can use ifcopenshell.util.element.get_decomposition.
